README.md

```diff
@@ -1,11 +1,9 @@
 # UCF FAQ CPT Plugin #
 
-Provides a custom post type for describing FAQ.
-
 
 ## Description ##
 
-Provides a custom post type for describing FAQ.
+Provides a custom post type for describing FAQs and supporting shortcodes. This theme supports the [UCF Athena Framework](https://github.com/UCF/Athena-Framework).
```

Comment:
The first sentence on this line is a bit confusing to me (and it might only be me), but I'm reading the second part as if it goes with the word 'describing', so "for describing supporting shortcodes". Should it be:
Provides a custom post type for describing FAQs and creates supporting shortcodes.